build.gradle

```
@@ -116,7 +117,7 @@ subprojects {
 
 ext {
     // NB: due to version.json generation by parsing this file, the next line must not have any if/then/else logic
-    neo4jVersion = "4.3.0-drop03.0"
+    neo4jVersion = "5.0.0-dev"
```

I was able to build it locally with this:
```suggestion
    neo4jVersion = "5.0.0-SNAPSHOT"
```

---

`5.0.0-dev` is needed for TeamCity. For local development we still need to override `neo4jVersionOverride`

---

This has been addressed in this commit: https://github.com/neo4j-contrib/neo4j-apoc-procedures/pull/2564/commits/8b3af787161d34d5fa854fc2bc6b7ec6f2a042d3